<launch>
    <!-- Start collecting IMU data and filter it -->
    <node name="joystick_sender" pkg="wurov" type="joystick_sender.py" args="--config_name Logitech">
        <rosparam file="$(find wurov)/config/controller.yaml" />
    </node>

     <!--<node name="joystick_sender" pkg="wurov" type="joystick_sender.py" args="- -config_name Xbox_test">
        <rosparam file="$(find wurov)/config/controller.yaml" />
    </node> 
    
    <node name="control_loop" pkg="wurov" type="pid_controller.py" args="- -dynamic_reconfig True"/>

    <include file="$(find wurov)/launch/imu_filter_madgwick.launch" /> -->

</launch>